crates/completion/src/completions.rs

```diff
@@ -13,6 +13,7 @@ pub(crate) mod postfix;
 pub(crate) mod macro_in_item_position;
 pub(crate) mod trait_impl;
 pub(crate) mod mod_;
+pub(crate) mod fuzzy_imports;
```

Comment:
:+1: 

I've been thinking about this for a long time. I thing a nice name here would be (on the) `flyimport`